src/main/webapp/WEB-INF/themes/root/resource/js/components/figshare.js

```diff
@@ -47,4 +71,28 @@ if (window.figshare) {
     window.loadedWidgets = loadedWidgets;
 
   });
+
+
+  // doubleclickad gets added to <aside> element of this widget. so rename it to <div>
```

Comment:
I want to understand this problem better.  What's an example article that I can use to see the issue?

Reply:
you can reproduce the issue with any article which has supporting information. Comment the function call and you can see the issue. I am attaching the screenshot also.

Reply:
<img width="707" alt="Screen Shot 2020-02-19 at 3 05 11 PM" src="https://user-images.githubusercontent.com/1896161/74885034-6ca8fc80-5329-11ea-9854-ba2902bf7f1f.png">